browser/src/react-redux/App/ForgotPasswordPage/ForgotPasswordPage.react.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+const initialState = {
+  email: '',
+  response: '',
+};
+
+class ForgotPasswordPage extends Component {
+  state = initialState;
+
+  handleChange = (event) => {
+    this.setState({
+      [event.target.name]: event.target.value,
+    });
+  }
+
+  onSubmit = (e) => {
+    e.preventDefault();
+    this.setState({ response: '' });
+    const { email } = this.state;
+    const userEmail = {
+      email,
+    };
+    axios
+      .post('/api/auth/forgot', userEmail)
+      .then(() => {
+        this.setState({ response: 'You should receive an email, soon, with instructions to reset your password.' });
+      })
+      .catch((error) => {
+        const { message } = error.response.data;
+        this.setState({ response: message });
+      });
+  }
+
+  render() {
+    const { email, response } = this.state;
+    return (
+      <div className="forgot-page">
+        <form onSubmit={this.onSubmit} className="forgot-page__form">
```

Comment:
At work, I've been introduced to breaking props on their own line when you have more than 1 prop. I think it looks a lot cleaner. I'd like to bring that convention into this codebase.

So this would turn into:
```
<form
  onSubmit={this.onSubmit}
  className="forgot-page__form"
>
```